packages/open-next/src/adapters/logger.ts

```diff
@@ -42,28 +42,33 @@ const isDownplayedErrorLog = (errorLog: AwsSdkClientCommandErrorLog) =>
 
 export function error(...args: any[]) {
   // we try to catch errors from the aws-sdk client and downplay some of them
-  if (
-    args.some((arg: AwsSdkClientCommandErrorLog) => isDownplayedErrorLog(arg))
-  ) {
+  if (args.some((arg) => isDownplayedErrorLog(arg))) {
     debug(...args);
-  } else if (args.some((arg) => arg.__openNextInternal)) {
+  } else if (args.some((arg) => isOpenNextError(arg))) {
     // In case of an internal error, we log it with the appropriate log level
-    const error = args.find(
-      (arg) => arg.__openNextInternal,
-    ) as BaseOpenNextError;
-    if (error.logLevel === 0) {
-      debug(...args);
+    const error = args.find((arg) => isOpenNextError(arg))!;
+    if (error.logLevel < getOpenNextErrorLogLevel()) {
       return;
     }
+    if (error.logLevel === 0) {
+      // Display the name and the message instead of full Open Next errors.
+      return debug(
```

Comment:
We should probably add some kind of warning just before this line in case `openNextDebug` is not set, otherwise there will be no error logged and it might be confusing for people.
Or we just use `console.log` or `console.error` and rely entirely on `OPEN_NEXT_ERROR_LOG_LEVEL`

Reply:
Good idea.
+1 to drop openNextDebug